crates/sovereign-sdk/full-node/db/sov-db/src/rocks_db_config.rs

```diff
@@ -67,22 +67,19 @@ impl<'a> RocksdbConfig<'a> {
         // Default is Snappy but Lz4 is recommend
         // https://github.com/facebook/rocksdb/wiki/Compression
         db_opts.set_compression_type(rocksdb::DBCompressionType::Lz4);
-        db_opts.set_compression_options_parallel_threads(4);
+        db_opts.set_compression_options_parallel_threads(2);
+
+        db_opts.increase_parallelism(2);
 
         db_opts.set_max_open_files(self.max_open_files);
         db_opts.set_max_total_wal_size(self.max_total_wal_size);
         db_opts.set_max_background_jobs(self.max_background_jobs);
         if !readonly {
-            // Increase write buffer size to reduce allocations.
-            db_opts.set_write_buffer_size(30 * 1024 * 1024); // 30 MB
```

Comment:
Reasonable, i checked the default it was indeed 64... i must have mistakenly found the default to be way less than that when i set this.